tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -134,5 +146,18 @@ def _get_num_of_partitions(self, key: ShelveKey) -> int:
         except self.client.exceptions.ResourceNotFoundException:
             return 0
 
+    def __setitem__(self, key: ShelveKey, val: bytes) -> None:
```

Comment:
can you move this above _delete_item to reduce the diff and avoid missing unintentional changes?